pkg/cli/scan.go

```diff
@@ -614,6 +614,7 @@ func resolveInputFileFromArg(inputFilePath string) (*os.File, error) {
 func resolveInputForRemoteTarget(ctx context.Context, input string) (downloadedAPKFilePaths []string, cleanup func() error, err error) {
 	logger := clog.FromContext(ctx)
 
+	noPackagesFound := 0
 	for _, arch := range []string{"x86_64", "aarch64"} {
```

Comment:
This looks like it will work today, I just have on thought to make it less likely to break in the future — could we invert this and say `archesFound := 0`, and then increment it when we **_do find_** a package for that architecture. Down below, we could have the error condition be `if archestFound == 0` instead of `if noPackagesFound == 2`. WDYT? I don't feel strongly, it just seems a bit more straightforward to my brain

Reply:
sound great that!
ptal